fastlane_bot/helpers/routehandler.py

```diff
@@ -320,7 +320,7 @@ def handle_custom_data_extras(self, platform_id: int, custom_data: bytes, exchan
 
         assert custom_data in "0x", f"[routehandler.py handle_uni_v3_router_switch] Expected the custom data field to contain '0x', but it contained {custom_data}. This function may need to be updated."
         if platform_id == self.ConfigObj.network.EXCHANGE_IDS.get(self.ConfigObj.network.UNISWAP_V3_NAME):
-            if self.ConfigObj.network.NETWORK in "ethereum" or exchange_name in self.ConfigObj.network.PANCAKESWAP_V3_NAME:
+            if self.ConfigObj.network.NETWORK in "ethereum" or exchange_name in [self.ConfigObj.network.PANCAKESWAP_V3_NAME, self.ConfigObj.network.BUTTER_V3_NAME, self.ConfigObj.network.AGNI_V3_NAME, self.ConfigObj.CLEOPATRA_V3_NAME]:
```

Comment:
I would suggest changing the conditional logic to `x=="ethereum"` instead of `x in "ethereum"`. On a related note, it would be better to define "ethereum" in constants.py, however it may not prudent to change this here and not everywhere else... so for the time being it's OK but just want to call it out for future reference.

Reply:
Updated this - could you please look again?